install_cov_tools.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cargo install rustfilt
+rustup component add llvm-tools-preview
+cargo install cargo-binutils
```

Comment:
can also add `cargo install ripgrep` for `rg` to work, unless everyone is supposed to have it :)